Sources/OpenCombine/OpenCombine.docc/Info.plist

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CFBundleName</key>
+	<string>OpenCombine</string>
+	<key>CFBundleDisplayName</key>
+	<string>OpenCombine</string>
+	<key>CDDefaultCodeListingLanguage</key>
+	<string>swift</string>
```

Comment:
This file is needed. Because we are using some doc formats like the following ⬇️
```
/// A protocol for uniquely identifying publisher streams.
///
/// If you create a custom ``Subscription`` or ``Subscriber`` type, implement this protocol so that development tools can uniquely identify publisher chains in your app.
/// If your type is a class, Combine provides an implementation of ``CustomCombineIdentifierConvertible/combineIdentifier-2xc6y`` for you.
/// If your type is a structure, set up the identifier as follows:
///
///     let combineIdentifier = CombineIdentifier()
```

We need to add this info.plist file to make it render as a Swift code block.